docs/manual/common/concepts/InternalAndExternalCommunication.md

```diff
@@ -0,0 +1,7 @@
+#Internal and external communication
+
+Lagom promotes use of asynchronous communication without preventing use of synchronous communication where necessary. For example, as mentioned in [[Polyglot systems with Lagom|PolyglotSystems]], legacy application components might use synchronous communication that cannot be changed.  When choosing the mode of communication for your microservices, keep in mind inter-service communication as well as external communication. 
```

Comment:
When I removed the sentence there was not much left. :-( How about this to replace it:
Within your system, services might communicate with other services in the same node and services in other nodes. And, I like Ignasi's suggestion. I would like to do it in this PR, if one of you can write up three bullets.